TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TwoControllerDriverControlled.java

```diff
@@ -15,49 +13,42 @@ public void runOpMode() throws InterruptedException {
         Robot robot = new Robot(hardwareMap, time);
         double x, y, r, target, targetAngle;
 
-        // Debouncers
+        //Debouncers
         Debouncer dx = new Debouncer(),
         bumper = new Debouncer(),
         clawdebouncer = new Debouncer(),
-        turnAroundDebouncer = new Debouncer();
-
-        boolean fieldCentric = false;
-        LiftControlMode liftControlMode = LiftControlMode.ManualControl;
+        turnAroundDebouncer = new Debouncer(),
+        gyroAssistDebouncer = new Debouncer();
 
-        //Quickly tweak sensitivity coefficients here
-        double gpad1MoveSpeed = .9;
+        //Coefficients
+        double gpad1MoveSpeed = 1;
         double gpad1RotationSpeed = 0.7;
         double gpad2MoveSpeed = 0.0;
         double gpad2RotationSpeed = 0.0;
-        double tempTarget = 0;
-
-        // Exp for smoothing joystick values
         double joystickSmoothingExp = 1.6;
-
-
-        //negative coefficient to account for backwards lift control. Invert as needed.
+        //negative lift coefficient. Change as needed.
```

Comment:
maybe say change value as needed. just to make sure the user does not change the lift's sign